lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -5524,6 +5524,47 @@ class ConvertAtenOpToTosaCustomOp : public OpConversionPattern<AtenOpT> {
   std::string implementedWithOpAttr;
 };
 
+template <>
+class ConvertAtenOpToTosaCustomOp<AtenGeluOp>
+    : public OpConversionPattern<AtenGeluOp> {
+public:
+  using OpConversionPattern<AtenGeluOp>::OpConversionPattern;
+  using OpAdaptor = typename AtenGeluOp::Adaptor;
+
+  ConvertAtenOpToTosaCustomOp(TypeConverter &typeConverter,
+                              MLIRContext *context, std::string /*opName*/,
+                              std::string /*implementedWithOpAttr*/)
```

Comment:
Isn't `implementedWithOpAttr` already `"UNDEF"` and you can use this later on for `implement_attrs`?

Reply:
We would need to store it somewhere, but for this specific customOp gelu, we don't the Conversion pass to store anything.  =)